x/gravity/types/msgs.go

```diff
@@ -4,9 +4,10 @@ import (
 	"encoding/hex"
 	"fmt"
 
+	sdkerrors "cosmossdk.io/errors"
 	"github.com/cometbft/cometbft/crypto/tmhash"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	errorsmod "github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
We should use `cosmossdk.io/errors` for errors

Reply:
updated by @ruthishvitwit